content/docs/bounties.md

```diff
@@ -44,3 +44,86 @@ As of 05/18/2024, the total payout for this bounty is **0.011 BTC**.
   - This existing crate could be a useful starting point, but is not required to be used for this bounty.
 
 {{% /details %}}
+
+{{% details title="Implement sending to Silent Payment addresses in Sparrow - **0.01 BTC**" closed="true" %}}
+
+### Completion requirements
+
+A merged pull request against the Sparrow repository, implementing sending to Silent Payment addresses.
+
+### Contributors
+
+| Person | Amount |
+| --- | --- |
+| Anon | 1000k sats |
+
+### Relevant links
+
+- [Sparrow Github](https://github.com/sparrowwallet/sparrow)
+- [Existing Rust crate for Silent Payments support](https://github.com/cygnet3/rust-silentpayments)
+  - This existing crate could be a useful starting point, but is not required to be used for this bounty.
+
+{{% /details %}}
+
+{{% details title="Implement sending to Silent Payment addresses in Samourai - **0.01 BTC**" closed="true" %}}
+
+### Completion requirements
+
+A merged pull request against whatever will be the active Samourai repository, implementing sending to Silent Payment addresses.
+
+### Contributors
+
+| Person | Amount |
+| --- | --- |
+| Anon | 1000k sats |
+
+### Some notes
+
+Given the recent events, some people may have concerns about this bounty, but Samourai being code is in the end nothing more than speech, ie. an idea, and what it stands for is still as true and valuable as ever, people can uphold it for a time, other people can and must uphold it if they also believe in it. We are very much aware of previous antagonisms and online flame wars especially on this issue, and this bounty is not a provocation either, we just want the idea of bitcoin privacy to keep spreading and think that this bounty makes our point.
+
+### Relevant links
+
+- [Existing Rust crate for Silent Payments support](https://github.com/cygnet3/rust-silentpayments)
```

Comment:
I see this crate is mentioned under all of them, is it just a placeholder? I know there are better options for BlueWallet at least, and I know that Samourai is not Rust-based.

Reply:
Yes it was mostly a placeholder, if you have suggestions for more relevant links we must update it. I still think that mentioning the rust crate is useful at least for reference, and projects written in other languages could even use it directly, it all depends on each project specifics of course as it is not necessarily the most practical approach but there are ffi to many languages. Regarding Samourai in particular I already did some research last year and I think it's possible to use the rust crate, even if a bit annoying.